src/utils/monitor.js

```diff
@@ -1,2 +1,4 @@
+import { LIMIT } from 'src/const/config';
+
 // eslint-disable-next-line import/prefer-default-export
-export const DEFAULT_LIMIT = 20;
+export const DEFAULT_LIMIT = LIMIT; // deprecated
```

Comment:
deprecated?

Reply:
moved to config we shouldn't use it from here and will remove at the end of this objective